x-pack/plugins/cases/public/components/user_list/index.tsx

```diff
@@ -34,48 +34,51 @@ interface UserListProps {
   users: ElasticUser[];
 }
 
-const MyAvatar = styled(EuiAvatar)`
-  top: -4px;
-`;
-
-const MyFlexGroup = styled(EuiFlexGroup)`
-  ${({ theme }) => css`
-    margin-top: ${theme.eui.euiSizeM};
-  `}
-`;
-
 const renderUsers = (
   users: ElasticUser[],
   handleSendEmail: (emailAddress: string | undefined | null) => void
 ) =>
   users.map(({ fullName, username, email }, key) => (
-    <MyFlexGroup key={key} justifyContent="spaceBetween" responsive={false}>
-      <EuiFlexItem grow={false}>
-        <EuiFlexGroup gutterSize="xs" responsive={false}>
+    <div key={key}>
+      <EuiFlexGroup key={key} justifyContent="spaceBetween" alignItems="center" responsive={false}>
+        <EuiFlexItem grow={false}>
+          <EuiFlexGroup responsive={false} alignItems="center" gutterSize="s">
+            <EuiFlexItem grow={false}>
+              <EuiAvatar name={fullName ? fullName : username ?? ''} size="s" />
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              {fullName ? (
+                <EuiToolTip position="top" content={<p>{fullName ? fullName : username ?? ''}</p>}>
+                  <EuiText size="xs">
+                    <p>
+                      <span data-test-subj="case-view-username">{username}</span>
+                    </p>
+                  </EuiText>
+                </EuiToolTip>
+              ) : (
+                <EuiText size="xs">
+                  <p>
+                    <span data-test-subj="case-view-username">{username}</span>
+                  </p>
+                </EuiText>
+              )}
+            </EuiFlexItem>
+          </EuiFlexGroup>
+        </EuiFlexItem>
+        {isEmpty(email) === false && (
```

Comment:
There is a bug that I think existed in this view where `email` ALWAYS returns `null` of an empty string, even when set. All i did was decide not to show the button icon if there's no email, where before it would show a disabled button. This needs to get solved deeper in the code and likely will require engineering help beyond this PR.